cmd/oras/push.go

```diff
@@ -181,5 +176,35 @@ func loadFiles(store *content.FileStore, annotations map[string]map[string]strin
 		}
 		files = append(files, file)
 	}
+	if len(files) == 0 {
+		fmt.Println("Uploading empty artifact")
+	}
 	return files, nil
 }
+
+func packManifest(ctx context.Context, store *file.Store, annotations map[string]map[string]string, opts *pushOptions) (ocispec.Descriptor, error) {
+	var packOpts oras.PackOptions
+	packOpts.ConfigAnnotations = annotations[annotationConfig]
+	packOpts.ManifestAnnotations = annotations[annotationManifest]
+	if opts.ManifestConfigRef != "" {
+		filename, mediaType := parseFileRef(opts.ManifestConfigRef, ocispec.MediaTypeImageConfig)
```

Comment:
Originally, `artifact.UnknownConfigMediaType` is used for the default config type. However, it is replaced by `ocispec.MediaTypeImageConfig`. Is it intended?